BoxBrowseSDK/BoxBrowseSDK/Categories/BOXItem+BOXBrowseSDKAdditions.m

```diff
@@ -0,0 +1,18 @@
+//
+//  BOXItem+BOXBrowseSDKAdditions.m
+//  BoxBrowseSDK
+//
+//  Created on 1/28/16.
+//  Copyright © 2016 BOX. All rights reserved.
+//
+
+#import "BOXItem+BOXBrowseSDKAdditions.h"
+
+@implementation BOXItem (BOXBrowseSDKAdditions)
+
+- (NSDate *)effectiveUpdateDate
+{
+    return self.isFile ? self.contentModifiedDate : self.modifiedDate;
```

Comment:
I think you want to check self.isFolder ? self.modifiedDate : self.contentModifiedDate